extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/options/HttpServerOptionsUtils.java

```diff
@@ -31,6 +31,21 @@
 @SuppressWarnings("OptionalIsPresent")
 public class HttpServerOptionsUtils {
 
+    /**
+     * When the http port is set to 0, replace it by this value to let Vert.x choose a random port
+     */
+    public static final int RANDOM_PORT_MAIN_HTTP = -1;
+
+    /**
+     * When the https port is set to 0, replace it by this value to let Vert.x choose a random port
+     */
+    public static final int RANDOM_PORT_MAIN_TLS = -2;
+
+    /**
+     * When the management port is set to 0, replace it by this value to let Vert.x choose a random port
+     */
+    public static final int RANDOM_PORT_MANAGEMENT = -3;
```

Comment:
Don't we need one for managment/TLS?

Reply:
No, management has only one port. Unlike the primary HTTP server which can serve both HTTP and HTTPS at the same time (using 2 ports), it's not possible with the management interface.

Reply:
Ah, okay.